### PR TITLE
add sql-query label to all k8s objects

### PR DESCRIPTION
**Pull request by chassing**
see [#APPSRE-5774](https://issues.redhat.com/browse/APPSRE-5774?focusedCommentId=20561862&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-20561862)

needed for #2522

### CODE REVIEW

File: reconcile/sql_query.py [hunk 1]
```diff
@@ -80,6 +80,8 @@
 kind: Job
 metadata:
   name: {{ JOB_NAME }}
+  labels:
+    sql-query: {{ LABEL }}
```
[the remark this answers is not in the document]
Comment (BumbleFeng) [reply reconcile/sql_query.py:84]: I suggest we use annotations by following all the openshift resources managed by other integrations
```
annotations:
    qontract.caller_name: {{ QUERY_NAME }}
    qontract.integration: QONTRACT_INTEGRATION
    qontract.integration_version: QONTRACT_INTEGRATION_VERSION
```

[the remark this answers is not in the document]
Comment (chassing) [reply reconcile/sql_query.py:84]: I'll add annotations, but we need labels to select the k8s objects afterward.

[the remark this answers is not in the document]
Comment (chassing) [reply reconcile/sql_query.py:84]: TIL [openshift_base.apply](https://github.com/app-sre/qontract-reconcile/blob/8014e56d21edf164340dc5663283c7e4773e7a97/reconcile/openshift_base.py#L320) is adding those annotations automatically :)

[the remark this answers is not in the document]
Comment (rporres) [reply reconcile/sql_query.py]: indeed, those annotations are added automatically but @chassing is right in pointing out that we need labels here too

If we want to use the same labels the integration uses plus a custom one for the query name, we can do:

```yaml
  labels:
    app: qontract-reconcile-sql-query
    component: qontract-reconcile
    query-name: {{ QUERY_NAME }}
```
    
It's also fine by me and will suit our purposes.

[the remark this answers is not in the document]
Comment (chassing) [reply reconcile/sql_query.py]: @BumbleFeng are you ok with the lastest changes?

**Review by rporres — APPROVED** [review]

**Pull request merged**